lib/core/engine.js

```diff
@@ -286,25 +286,7 @@ class Engine {
           result.statistics = statistics.summarizeDeep(options);
         }
       }).tap((result) => {
-        // don't bother if no statistics or silent x2
-        if (!result.statistics || !result.statistics.timings || !result.statistics.timings.pageTimings || options.silent > 1) return result;
-
-        let run = result.browserScripts[0].timings,
-          nRuns = result.browserScripts.length,
-          pt = result.statistics.timings.pageTimings,
-          t = result.statistics.timings,
-          m = nRuns > 1,
-          lines = [
-            `${run.resourceTimings.length + 1} requests`,
-            util.formatMetric('firstPaint', t.firstPaint, m),
-            util.formatMetric('DOMContentLoaded', pt.domContentLoadedTime, m),
-            util.formatMetric('Load', pt.pageLoadTime, m),
-            util.formatMetric('rumSpeedIndex', t.rumSpeedIndex, m),
-          ],
-          note = m ? ` (${nRuns} runs)` : '';
-
-        lines = lines.filter(Boolean).join(', ');
-        log.info(`${lines}${note}`);
+        log.info(util.getResultLogLine(result,options));
```

Comment:
I think the options.silent check belongs here, it's the callers responsibility. Additionally a check for an empty result, to avoid logging undefined/an empty row.